core/templates/pages/library-page/library-page.directive.html

```diff
@@ -50,7 +50,7 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
           <search-bar enable-dropup="false"></search-bar>
         </div>
 
-        <div class="oppia-classroom-div" ng-if="SHOW_CLASSROOM_CALLOUT">
+        <div class="oppia-classroom-div" ng-if="$ctrl.CLASSROOM_PAGE_IS_SHOWN">
```

Comment:
How it was working before without `$ctrl`?

Reply:
Before these changes, the constant was bound to `$scope` instead of `ctrl`.